content/03-reference/01-tools-and-interfaces/05-prisma-cli/02-command-reference.mdx

```diff
@@ -467,6 +467,99 @@ model Profile {
+
+- `db push` introspects the database, and infers and executes the changes required to make your database schema reflect the state of your Prisma schema.
+- By default, after changes have been applied to the database schema, generators are triggered (for example, Prisma Client). You do not need to manually invoke `prisma generate`.
+- If `db push` anticipates that the changes could result in data loss, it will:
+  
+  - Throw an error 
+  - Require the `—-force` option if you still want to make the changes
+
+- `db push` does not interact with or rely on migrations. The migrations table will no be updated, and no migration files will be generated. If migrations are detected, `db  push` will prompt for confirmation or require an explicit option (`--ignore-migrations`) 
+
+
+`db push` works well if:
+
+* You want to quickly prototype and iterate on schema design locally without the need to deploy these changes to other environments such as other developers, or staging and production environments.
+* You are prioritizing reaching a **desired end-state** and not the changes or steps executed to reach that end-state. There is no way to preview the changes that will be made to the database.
+* You do not need control over how schema changes impact data. There is no way to orchestrate schema and data migrations - if `db push` anticipates that changes will result in data loss, it will require specific confirmation via a `--force` option. You can either accept data loss by specifying the option or stop the process, but there is no way to customize the changes.
+
+`db push` is not recommended if:
+
+- You want to replicate your schema changes in other environments where there could be important data you cannot afford to lose. You can use `db push` for prototyping, but you should use migrations to commit the schema changes and apply these in your other environments.
+- You want fine-grained control over how the schema changes are executed - for example, renaming a column instead of dropping it and creating a new one.
+- You want to keep track of changes made to the database schema over time. `db push` does not create any artifacts that allow you to keep track of this information.
+- You want the schema changes to be reversible. You can use `db push` again to revert to the original state, but this might result in data loss and there will be no mechanism for you to orchestrate these changes.
+
+#### <inlinecode>db push</inlinecode> and Migrate (experimental)
+
+If you are already using migrations with Prisma Migrate, `db push` will still work. However, be aware of the following side effects of using both Prisma Migrate and `db push`:
+
+Prisma Migrate introspects the database to anticipate the changes needed to reach the end-state of the migration and preview these changes as DDL statements in [`README.md`](http://readme.md) files.
```

Comment:
http://readme.me link is probably here by accident. Is there any documentation we can point to for migrate that speaks about these files? Otherwise, I would remove.